File: src/main/java/com/websocket/demo/room/service/RoomService.java
package com.websocket.demo.room.service;

import com.websocket.demo.room.ChatRoom;
import com.websocket.demo.room.repository.RoomRepository;
import lombok.RequiredArgsConstructor;
import org.springframework.stereotype.Service;

import java.util.List;

@Service
@RequiredArgsConstructor
public class RoomService {

    private final RoomRepository roomRepository;

    public List<ChatRoom> findAll() {
        return roomRepository.findAll();
    }

    public ChatRoom findById(String id) {
        return roomRepository.findById(id);
    }

    public String create(String title) {
        return roomRepository.create(title);
    }
}

File: src/main/java/com/websocket/demo/room/service/RoomService.kt
package com.websocket.demo.room.service

import com.websocket.demo.room.ChatRoom
import com.websocket.demo.room.repository.RoomRepository
import org.springframework.stereotype.Service

@Service
open class RoomService(private val roomRepository: RoomRepository) {

    fun findAll(): List<ChatRoom> {
        return roomRepository.findAll()
    }

    fun findById(id: String?): ChatRoom {
        return roomRepository.findById(id!!)
    }

    fun create(title: String?): String {
        return roomRepository.create(title!!)
    }
}